apps/codeforafrica/src/payload/fields/links/link.js

```diff
@@ -0,0 +1,153 @@
+import { deepmerge } from "@mui/utils";
```

Comment:
Add an issue/story we can use to discuss how to reuse our payload fields effectively. This is one of the reason of using a monorepo after all.

Reply:
@kelvinkipruto [#571](https://github.com/CodeForAfrica/ui/issues/571)